README.md

```diff
@@ -19,59 +19,93 @@ usable child components and fast HTML rendering.
 
 To start with something small and simple we'll make a counter (see file `/examples/simple/simple.js`). First we need a place to store our data.
 
+`model.js`
 ```
-let model = {
+export default model = {
   count: 0,
 };
 ```
 
 Next we need rules to update our model
 
+`dispatcher.js`
 ```
-let update = [
-  ['inc', (params) => (state) => {
-    state.count += 1
-    return state
-  }],
-  ['dec', (params) => (state) => {
-    state.count -= 1
-    return state
-  }],
-]
+
+import {TanokDispatcher} from 'tanok';
+import {on} from 'tanok/lib/decorators';
+
+class CounterDispatcher extends TanokDispatcher {
+
+    @on('inc')
+    increment(eventPayload, state) {
+        state.count += 1;
+        return [state];
+    }
+
+    @on('dec')
+    decrement(eventPayload, state) {
+        state.count -= 1;
+        return [state];
+    }
+}
+
+export default CounterDispatcher;
 ```
 
 Last crucial thing for UI is it's visual representation
 
+`view.js`
 ```
-const Counter = React.createClass({
-  onPlusClick: function() {
-    this.props.eventStream.onNext({
-      action: 'inc',
-    })
-  },
-  onMinusClick: function() {
-    this.props.eventStream.onNext({
-      action: 'dec',
-    })
-  },
-  render: function() {
-        return <div>
-        <button onClick={this.onPlusClick}>+</button>
-        <span>{this.props.count}</span>
-        <button onClick={this.onMinusClick}>-</button>
-        </div>
+import TanokComponent from 'tanok/lib/component';
+
+class Counter extends React.Component {
+    constructor(props) {
+        super(props);
+        this.onPlusClick = this.onPlusClick.bind(this);
+        this.onMinusClick = this.onMinusClick.bind(this);
+    }
+
+    onPlusClick() {
+        this.send('inc');
+    }
+
+    onMinusClick() {
+        this.send('dec');
+    }
+
+    render() {
+        return (
+            <div>
+                <button onClick={this.onPlusClick}>+</button>
+                <span>
+                    {this.props.count}
+                </span>
+                <button onClick={this.onMinusClick}>-</button>
```

Comment:
I think that plain `onClick={() => this.send('dec')}` is more straightforward and therefore more suitable for a README example